notebooks/02-quality-flags/TIKEBook_Lesson2.ipynb

```diff
@@ -0,0 +1,780 @@
+{
```

Comment:
<p>I think we can be more specific for the first one... "Filter TESS lightcurve data based on quality flags and their meaning" ?</p><p> </p><p>Do we also want to include the more basic things being demonstrated here, like "Search the MAST archive by target name, download and view a lightcurve .fits file" and/or "Use the lightkurve package to search the MAST archive, download and view a lightcurve, create a periodogram, and view a folded light curve" ?</p><p> </p><p>Which brings up a larger point - do we want to just use one method? Or, make it clear that we are illustrating two different methods? (I think that can be valuable!)</p><p> </p><p>We can match the outline to the headers below, making sure the titles are descriptive and properly nested in a hierarchy.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Updated the opening text to include all of these things, made it clear we are illustrating two methods </p>